zk.js/src/test-utils/updateMerkleTree.ts

```diff
@@ -47,7 +48,18 @@ export async function updateMerkleTreeForTest(payer: Keypair, url: string) {
       transactionMerkleTree: transactionMerkleTreePda,
     });
   } catch (err) {
+    // TODO: Revisit recovery.
+    // Rn, we're just blanked-closing the update state account on failure which
+    // might not be desirable in some cases.
     console.error("failed at updateMerkleTreeForTest", err);
+    try {
+      console.log("closing update state account...");
+      await closeMerkleTreeUpdateState(merkleTreeProgram, payer, connection);
```

Comment:
what do you think about introducing another check in case the merkle tree update state creation fails?
(In catch check whether state already exists if yes close and retry.)